example/README

```diff
@@ -83,18 +83,24 @@ setsockopt(), writev(), sendfile64(), select(), epoll_create(), epoll_ctl(), epo
 ioctl() and fork().
 
 A script is provided in order to make utilization of this feature friendlier.
-
-# ./scripts/ofp_netwrap.sh  YOUR_NATIVE_APPLICATION
-
 Script's default configuration (see OFP_NETWRAP_ENV environment variable):
  - Interface eth1 is used and 192.168.100.1 is configured.
  - All cores (except for core 0) are used.
  - CLI file ./scripts/ofp_netwrap.cli is used for additional settings
 
+Execution:
+ - Set the path to the shared libraries.
+# export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:<Your path>/ofp/lib/.libs
```

Comment:
.libs at the end is probably unnecessary.

Reply:
It is needed:
Else
# ./scripts/ofp_netwrap.sh /home/bogdan/Work/test/in_udp
ERROR: ld.so: object 'libofp_netwrap_crt.so.0.0.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object 'libofp_netwrap_proc.so.0.0.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object 'libofp.so.0.0.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
Start:

Reply:
Interesting, on my system (Ubuntu 18.04) the complete opposite happens. E.g. `LD_LIBRARY_PATH=/home/nnn/ofp_install/lib/.libs` causes the aforementioned error and `LD_LIBRARY_PATH=/home/nnn/ofp_install/lib` works.

Reply:
That is because my path is in the 'Build folder' while yours is in the 'Install folder' (where you don't have a .libs). Point is the shared libs should be in load library path.

Reply:
That clears it. In my opinion it would be clearer to refer to the OFP library install path in this context.

Reply:
Added both.